### ISSUE TITLE
Slider extra colors (black and white)

### ISSUE THREAD

**Issue by sajen87** [xiaokaike/vue-color#118]
Hello,

I'm wondering if there is any chance to add black and white color to bottom tiles maybe by any chance??

**Comment by phivk** [comment]
`Slider.vue` defines the `swatches` array as part of the data function. You could include luminosity values 1 for white and 0 for black:

```
data () {
  return {
    swatches: ['1', '.65', '.50', '.35', '0']
  }
},
```

![screen shot 2018-04-24 at 10 03 00](https://user-images.githubusercontent.com/902958/39174072-e7ba20a6-47a6-11e8-8d5c-ac29c8002555.png)

@xiaokaike It might be nice if this could be controlled (via a prop?) when including the `slider-picker` component.

**Comment by linx4200** [comment]
fix in v2.6.0, which will be released soon.
